receiver/mongodbatlasreceiver/logs.go

```diff
@@ -0,0 +1,242 @@
+		tmp := strings.Split(s, ",")
+		for _, t := range tmp {
+
+			u, err := url.Parse(t)
+			if err != nil {
+				fmt.Printf("Error parsing out %s", t)
+			}
+
+			// separate hostname from scheme and port
+			host, _, err := net.SplitHostPort(u.Host)
+			if err != nil {
+				// the scheme prefix was not added on to this string
+				// thus the hostname will have been placed under the "Scheme" field
+				hostnames = append(hostnames, u.Scheme)
+			} else {
+				// hostname parsed successfully
+				hostnames = append(hostnames, host)
+			}
+
+		}
+	}
+
+	return hostnames
+}
+
+func (s *receiver) KickoffReceiver(ctx context.Context) {
+	stopper := make(chan struct{})
+	s.stopperChanList = append(s.stopperChanList, stopper)
+	cfgProjects, err := s.createProjectMap()
+	if err != nil {
+		s.log.Debug("error ")
+	}
+
+	for {
+		// reterive all organizations listed in the MongoDB client
+		orgs, err := s.client.Organizations(ctx)
+		if err != nil {
+			s.log.Debug("error retrieving organizations: %w")
+		}
+		// get all projects listed for each of the organizations
+		for _, org := range orgs {
+			projects, err := s.client.Projects(ctx, org.ID)
+			if err != nil {
+				s.log.Debug("error retrieving projects: %w")
+			}
+
+			// filter out any projects not specified in the config
+			filteredProjects := make(map[string]*mongodbatlas.Project)
+			for _, project := range projects {
+				if _, ok := cfgProjects[project.Name]; ok {
+					filteredProjects[project.Name] = project
+				}
+			}
+
+			// get clusters for each of the projects
+			for _, project := range filteredProjects {
+				resource := resourceInfo{Org: org, Project: project}
+				include, exclude := cfgProjects[project.Name].IncludeClusters, cfgProjects[project.Name].ExcludeClusters
+				clusters, err := s.client.GetClusters(ctx, project.ID, include, exclude)
+
+				// check to include or exclude clusters
+				switch {
+				//keep all clusters if include and exclude are not specified
+				case len(include) == 0 && len(exclude) == 0:
+					break
+				// include is initialized
+				case len(include) > 0 && len(exclude) == 0:
+					clusters, err = filterClusters(clusters, createStringMap(include), true)
+					break
+				// exclude is initialized
+				case len(exclude) > 0 && len(include) == 0:
+					clusters, err = filterClusters(clusters, createStringMap(exclude), false)
+					break
+				// both are initialized
+				default:
+					clusters = nil
+					s.log.Debug("Error can not have both include and exclude parameters initialized")
+				}
+
+				if err != nil {
+					s.log.Debug("error retreiving clusters from project")
+				}
+
+				// collection interval loop,
+				select {
+				case <-ctx.Done():
+					return
+				case <-stopper:
+					return
+				case <-time.After(collection_interval):
+					err = s.collectClusterLogs(clusters, cfgProjects, resource)
+					if err != nil {
+						s.log.Debug("Failure to collect logs from cluster")
```

Comment:
This should be an error log

Reply:
Also please include the error as a field in this log